install/kubernetes/helm/istio/charts/kiali/templates/service.yaml

```diff
@@ -10,6 +10,5 @@ spec:
   - name: tcp
     protocol: TCP
     port: 20001
-    name: http-kiali
```

Comment:
Please remove the name of `name: tcp` and keep `name: http-kiali`, otherwise, the service will not be traced by Jaeger. Please refer to https://istio.io/docs/setup/kubernetes/spec-requirements/ for detail.